example/rollup/rollup.config.js

```diff
@@ -1,11 +1,11 @@
 import commonjs from 'rollup-plugin-commonjs';
-import resolve from 'rollup-plugin-node-resolve';
+import resolve from '@rollup/plugin-node-resolve';
 import {terser} from 'rollup-plugin-terser';
 
 // The html plugin generates a file that M::P::Webpack reads to generate the
 // correct output when calling <%= asset 'example.js' %> and
 // <%= asset 'example.css' %>
-import html from 'rollup-plugin-bundle-html';
+const html = require('@rollup/plugin-html');
```

Comment:
Why is this changed from `import` to `const`?

Reply:
An in haste copy-paste.